### DISCUSSION TITLE
NCBI Prok not functioning on MiGA web

### DISCUSSION THREAD

**Discussion by cldrozd** [bio-miga/miga#181] [General]
Hello,

I have been having issues with using the NCBI Prok tool on MiGA web. When I submit a genome, it just seems to process endlessly, never producing a result. This happens on any genome I submit, even ones that previously had worked. Is there a fix for this? 

Thank you!

**Comment by lmrodriguezr** [comment]
Dear @cldrozd 
Unfortunately the gatech servers have been misbehaving for quite some time, so I would recommend using instead:
https://lfu.microbial-genomes.org/

Best wishes, Miguel.
